.env.example

```
@@ -0,0 +1,15 @@
+# 애플리케이션 설정
+PROJECT_NAME=FastAPI Community
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)